naslib/utils/utils.py

```diff
@@ -322,8 +322,8 @@ def get_train_val_loaders(config, mode="train"):
     data = config.data
     dataset = config.dataset
     seed = config.search.seed
-    batch_size = config.batch_size
-    train_portion = config.train_portion
+    batch_size = config.batch_size if hasattr(config, "batch_size") else config.search.batch_size
```

Comment:
This looks like a bug fix independent of the visualization code. If so, create a new PR, or push the fix directly to Develop.